compiler/rustc_codegen_cranelift/src/common.rs

```diff
@@ -61,8 +63,10 @@ fn clif_type_from_ty<'tcx>(tcx: TyCtxt<'tcx>, ty: Ty<'tcx>) -> Option<types::Typ
         },
         ty::Char => types::I32,
         ty::Float(size) => match size {
+            FloatTy::F16 => unimplemented!("f16 is not yet implemented"),
             FloatTy::F32 => types::F32,
             FloatTy::F64 => types::F64,
+            FloatTy::F128 => unimplemented!("f128 is not yet implemented"),
```

Comment:
Won't this cause panics when compiling the standard library?

Reply:
I was assuming not because the cranelift tests seem to still pass when I run locally. But I don't know what the exact combination of rustc, cranelift, and bootstrap settings is.

Is there something better to do here? If you have a patch I will apply it during my next rebase, or I can make the change if it's something easy.

@antoyo I do a similar thing for cg_gcc, what do you suggest there?

Reply:
cg_clif inside the rust repo reuses the llvm built standard library but the standalone repo uses builds the standard library using cg_clif. cg_gcc can't reuse the llvm built standard library as it depends on all crates being compiled with `-Csymbol-mangling-version=v0` which is not the default.

Reply:
I guess if the CI for cg_gcc indeed fails, you could try using the supported types (F32, F64) and I'll fix this on my side when doing the next sync.
If the CI still fails, I can add the missing stuff to libgccjit: it shouldn't be that hard.